src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
 * Deprecated `SimpleAuthenticatorInterface`, `SimpleFormAuthenticatorInterface`,
   `SimplePreAuthenticatorInterface`, `SimpleAuthenticationProvider`, `SimpleAuthenticationHandler`,
   `SimpleFormAuthenticationListener` and `SimplePreAuthenticationListener`. Use Guard instead.
+* dispatch a `VoteEvent` when a voter makes a vote 
```

Comment:
(only in debug mode)

Reply:
not sure it should even be mentioned as it's internal

Reply:
Right ! Thank you :)